### PR TITLE
Add admin users

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: cmd/admin-users.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * limitations under the License.
+ */
+
+package cmd
+
+import "github.com/minio/cli"
+
+var adminUsersCmd = cli.Command{
+	Name:   "users",
+	Usage:  "Manage mulitple users",
```
Comment (poornas) [inline cmd/admin-users.go]: typo here on multiple - Manage users would be better

**Comment by harshavardhana** [comment]
This PR is changing a bit

File: docs/minio-admin-complete-guide.md [hunk 2]
```diff
@@ -273,19 +275,69 @@ FLAGS:
   Storage : Used 8.2GiB
 ```
 
+<a name="users"></a>
+### Command `users` - Manage regular users
+`users` command to add new, remove, enable, disable, list users on Minio server.
```
Comment (poornas) [inline docs/minio-admin-complete-guide.md]: add new, -> add

File: cmd/admin-users-add.go [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+	Name:   "add",
+	Usage:  "Add new users",
+	Action: mainAdminUsersAdd,
+	Before: setGlobalsFromContext,
+	Flags:  globalFlags,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} TARGET USERNAME PASSWORD POLICYTYPE|POLICYFILE
```
Comment (vadmeste) [inline cmd/admin-users-add.go]: Can we add possible values of policy type ? and a short example of policy file content

**Comment by vadmeste** [comment]
One little problem is when trying to upload an object, mc cp, in a bucket.. it doesn't work because for some reasons, mc want to list all buckets, readwrite policy won't guarantee it that, but users can use custom policy document probably and then fix this later

**Comment by harshavardhana** [comment]
> One little problem is when trying to upload an object, mc cp, in a bucket.. it doesn't work because for some reasons, mc want to list all buckets, readwrite policy won't guarantee it that, but users can use custom policy document probably and then fix this later

This is something of a problem in `minio-go` I think it constructs policies with explicit API names but not for everything.

**Pull request merged**